DLTtxRecipt/schema.json

```diff
@@ -0,0 +1,164 @@
+                            "maxLength": 256,
+                            "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$",
+                            "description": "Property. Identifier format of any NGSI entity"
+                        },
+                        {
+                            "type": "string",
+                            "format": "uri",
+                            "description": "Property. Identifier format of any NGSI entity"
+                        }
+                    ],
+                    "description": "Relationship. Model:'http://schema.org/URL'. Entity presisted in the DLT"
+                },
+                "TxRecipts": {
+                    "type": "object",
+                    "description": "Property. Transaction Recipt",
+                    "properties": {
+                        "to": {
+                            "type": "string",
+                            "description": "Property. Model:'https://schema.org/Text'. Account or Contract Address to transaction has been submitted"
+                        },
+                        "from": {
+                            "type": "string",
+                            "description": "Property. Model:'https://schema.org/Text'. Account Address of the user/service responsible to submit the transaction (address of the msg.sender)"
+                        },
+                        "keys": {
+                            "type": "array",
+                            "description": "Property. Payload keys used in transaction payload",
+                            "items": {}
+                        },
+                        "logs": {
+                            "type": "object",
+                            "description": "Property. A log record can be used to describe an event within a smart contract (Ethereum)",
+                            "properties": {
+                                "id": {
+                                    "type": "string",
+                                    "description": "Property. Model:'https://schema.org/Text'. log id"
+                                },
+                                "data": {
+                                    "type": "string",
+                                    "description": "Property. Model:'https://schema.org/Text'. contains one or more 32 Bytes non-indexed arguments of the log"
+                                },
+                                "topics": {
+                                    "type": "object",
+                                    "description": "Property. Array of 0 to 4 32 Bytes DATA of indexed log arguments. (In solidity: The first topic is the hash of the signature of the event (e.g. Deposit(address,bytes32,uint256)), except you declared the event with the anonymous specifier.)",
+                                    "items": {}
+                                },
+                                "address": {
```

Comment:
It would be better not to use address because it could lead to confusion with the first level property, better something like logAddress